pages/api/create-order.ts

```diff
@@ -24,7 +24,13 @@ export default async function handler(
       total,
       items,
       customerId,
+      vendor: {
+        foo: "bar",
```

Comment:
More closely mimicks what is passed in
```suggestion
        id: "foo",
```

Reply:
I think foo is a better indicate it can be anything